app/src/main/java/com/azuresamples/msalandroidapp/SingleAccountModeFragment.java

```diff
@@ -163,22 +164,13 @@ public void onError(@NonNull MsalException exception) {
             }
         });
 
-        callGraphApiInteractiveButton.setOnClickListener(new View.OnClickListener() {
+        signInAgainButton.setOnClickListener(new View.OnClickListener() {
             public void onClick(View v) {
                 if (mSingleAccountApp == null) {
                     return;
                 }
 
-                /**
-                 * If acquireTokenSilent() returns an error that requires an interaction (MsalUiRequiredException),
```

Comment:
let's keep this comment for the developers.

Reply:
(replacing acquireToken() with signInAgain() ofc)

Reply:
Updated!